apps/qubit/modules/settings/templates/diacriticsSuccess.php

```diff
@@ -0,0 +1,80 @@
+<?php decorate_with('layout_2col.php'); ?>
+
+<?php slot('sidebar'); ?>
+
+<?php echo get_component('settings', 'menu'); ?>
+
+<?php end_slot(); ?>
+
+<?php slot('title'); ?>
+<h1>
+  <?php echo __('Diacritics settings'); ?>
+</h1>
+<?php end_slot(); ?>
+
+<?php slot('content'); ?>
+
+<div class="alert alert-info">
+  <p>
+    <?php echo __('Please rebuild the search index after uploading diacritics mappings.'); ?>
+  </p>
+  <pre>$ php symfony search:populate</pre>
+</div>
+
+<div class="alert alert-info">
+  <p>
+    <?php echo __('Example CSV:'); ?>
+  </p>
+  <pre>type: mapping<br/>mappings:<br/>  - À => A<br/>  - Á => A</pre>
+</div>
+
+<?php echo $form->renderGlobalErrors(); ?>
+
+<?php echo $form->renderFormTag(url_for(['module' => 'settings', 'action' => 'diacritics']), ['method' => 'post']); ?>
+
+<?php echo $form->renderHiddenFields(); ?>
+
+<div id="content">
+
+  <table class="table sticky-enabled">
+    <thead>
+      <tr>
+        <th width="30%">
+          <?php echo __('Name'); ?>
+        </th>
+        <th>
+          <?php echo __('Value'); ?>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <?php echo __('Diacritics'); ?>
+        </td>
+        <td>
+          <?php echo $form->diacritics; ?>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <?php echo __('Mappings YAML'); ?>
+        </td>
+        <td>
+          <span>
+            <?php echo __('YAML file containing char_filter mappings.'); ?>
+          </span></br>
+          <?php echo $form->mappings; ?><br />
+        </td>
+      </tr>
+    </tbody>
+  </table>
```

Comment:
To display error messages like the media type validation error, the fields need to be rendered with:

```php
<?php echo $form->mappings->label(__('Mappings YAML'))->renderRow(); ?>
```

You may be able to use the help message for the span text and I'd suggest to not use the table display and use something similar to the `csvValidatorSuccess.php` template.